### PR TITLE
Fix defaultactgrp

### PR DESCRIPTION
**Pull request by lanarimarco**
## Description

The default activation name is passed through a non-mandatory property: Configuration.defaultActivationGroupName.

## Checklist:
- [X] There are tests regarding this feature
- [X] The code follows the Kotlin conventions (run `./gradlew ktlintCheck`)
- [X] The code passes all tests (run `./gradlew check`)
- [] There is a specific documentation in the `docs` directory

### CODE REVIEW

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/execution/Configuration.kt [hunk 1]
```diff
@@ -9,22 +9,33 @@ import com.smeup.rpgparser.interpreter.ISymbolTable
  * @param memorySliceStorage Allows to implement a symbol table storaging.
  * If null, symbol table persistence will be skipped
  * @param jarikoCallback Several callback.
+ * @param defaultActivationGroupName Default activation group. If not specified it assumes "*DEFACTGRP"
  * */
+
+const val DEFAULT_ACTIVATION_GROUP_NAME = "*DEFACTGRP"
```
Comment (benetti-smeup) [inline rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/execution/Configuration.kt:15]: It should be *DFTACTGRP

**Review by benetti-smeup — CHANGES_REQUESTED** [review]
*DEFACTGRP should be *DFTACTGRP

**Comment by lanarimarco** [comment]
> *DEFACTGRP should be *DFTACTGRP

**Comment by lanarimarco** [comment]
> *DEFACTGRP should be *DFTACTGRP

Ok

**Review by benetti-smeup — APPROVED** [review]
ok

**Pull request merged**